### PR TITLE
Fix/73/code clean up

### PR DESCRIPTION
**Pull request by Dmitruk89**
1. rewrite eslint rule for unused-vars. Now it doesnt work in constructor, but if it is really unused you get an error.
2. fix all the warnings
3. remove some commented code

### CODE REVIEW

**Comment by Dmitruk89** [comment]
4. rewrite eslint rule array-element-newline.

**Review by PaulRomanov — APPROVED** [review]

**Review by Nicolai-boiko — APPROVED** [review]

File: src/app/components/login-form/login-form.component.ts [hunk 1]
```diff
@@ -17,10 +18,10 @@ export class LoginFormComponent {
...
   submit(): void {
     if (this.email.valid && this.password.valid) {
-      const user = {
-        username: this.email.value,
-        password: this.password.value,
-      };
+      // const user = {
+      //   username: this.email.value,
+      //   password: this.password.value,
+      // };
```
Comment (mhasilau) [inline src/app/components/login-form/login-form.component.ts:24]: delete comments

File: src/app/services/api.service.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export abstract class ApiService<T extends ResourceModel<T>> {
     } else {
       // The backend returned an unsuccessful response code.
       // The response body may contain clues as to what went wrong,
-      console.error(`Backend returned code ${err.status}, body was: ${err.error}`);
+      console.error(`Backend returned code ${err.status}, body was: ${String(err.error)}`);
     }
 
     // ...optionally return a default fallback value so app can continue (pick one)
```
Comment (mhasilau) [inline src/app/services/api.service.ts:59]: delete comments

File: src/app/services/api.service.ts [hunk 3]
```diff
@@ -53,7 +53,7 @@ export abstract class ApiService<T extends ResourceModel<T>> {
     } else {
       // The backend returned an unsuccessful response code.
       // The response body may contain clues as to what went wrong,
```
Comment (mhasilau) [inline src/app/services/api.service.ts:55]: delete comments

**Review by mhasilau — APPROVED** [review]

File: src/app/services/api.service.ts (hunk 3 above)
Comment (vstrabko) [reply src/app/services/api.service.ts:55]: those comments are OK, as they clarify why this part of the code is needed.

File: src/app/services/api.service.ts (hunk 2 above)
Comment (vstrabko) [reply src/app/services/api.service.ts:59]: same here

**Review by vstrabko — APPROVED** [review]

File: src/app/components/login-form/login-form.component.ts (hunk 1 above)
Comment (Dmitruk89) [reply src/app/components/login-form/login-form.component.ts]: deleted.

**Pull request merged**